### ISSUE TITLE
Paramount Plus Movies

### ISSUE THREAD

**Issue by FortniteballsXD** [ytdl-org/youtube-dl#29223]
## Checklist

- [x ] I'm reporting a site feature request
- [x ] I've verified that I'm running youtube-dl version **2021.06.06**
- [ x] I've searched the bugtracker for similar site feature requests including closed ones

## Description

As we all know Paramount Plus was added but the problem is that it doesn't do movies but any tv show you throw at it works I feel that we should add movie support because its the same thing as shows but instead of paramountplus.com/shows its paramountplus.com/movies

**Comment by Danker1990** [comment]
Yes but on this occasion DRM is attached to all movies placed on there. Shows will untill a drastic redesign will download whereas movies will not.

**Comment by 89z** [comment]
https://github.com/ytdl-org/youtube-dl/issues/30491

**Comment by 89z** [comment]
Issue should be closed, no URL provided
